### DISCUSSION TITLE
BIRT Y

### DISCUSSION THREAD

**Discussion by webtrees-pesz** [FamilySearch/GEDCOM#131] [Q&A] [answered]
GEDCOM allows BIRT Y to indicate that a birth has occurred. 
In what cases is this useful? Are there any examples? Is it possible to have a person without a birth?

Best regards
Peter

**Comment by tychonievich** [comment, marked as answer]
There are researchers who want to create INDI records for fetuses without creating a BIRT for them. But in the common case I think most people assume that the following two are equivalent:

```gedcom
0 @I1@ INDI
```

```gedcom
0 @I1@ INDI
1 BIRT Y
```

Note that we kept `BIRT Y` in 7.0 because it was in 5.5.1 and we saw no compelling reason to remove it.
